DeceptionGame/Assets/Scripts/MapGeneration/ShowOnlyIfInRange.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ShowOnlyIfInRange : MonoBehaviour
```

Comment:
Hide a player if they are a certain distance away by changing the layer and culling it. The reason why this is used is because I still wanted the map under the fog of war to be slightly visible if you visited it. There might be a better way to do this.